docs/walkthrough/isolated-functions.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Isolated Functions
+nav_order: 13
+parent: Getting Started
+---
+
+### Isolated Functions
+
+An isolated function is written for a specific purpose. It requires an input, presents an output, and is capable of returning error information in the event that anything goes worong - not unlike regular isolates. The key difference between an isolated function and a traditional isolate is that an isolated function is designed to only ever run once, whereas a traditional isolate is written as a reusable server.
```

Comment:
Maybe instead of "The key difference between an isolated function and a traditional isolate is that an isolated function is designed to only ever run once, whereas a traditional isolate is written as a reusable server.":

"An isolated function is booted up, performs its task, and then immediately terminates. Compared to other function types this is a simple and straight-forward design, but it comes at a cost to performance and latency.

Something like that?